data/ui/oops-window.ui

```diff
@@ -1,247 +1,98 @@
-                    <property name="image">image2</property>
-                  </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="fill">True</property>
-                    <property name="position">2</property>
-                  </packing>
-                </child>
-                <child>
-                  <object class="GtkToggleButton" id="tbtn_multi_select">
-                    <property name="visible">True</property>
-                    <property name="can_focus">True</property>
-                    <property name="receives_default">True</property>
-                    <property name="image">image1</property>
-                    <property name="use_underline">True</property>
-                    <signal name="toggled" handler="on_tbtn_multi_select_toggled" swapped="no"/>
-                  </object>
-                  <packing>
-                    <property name="position">3</property>
-                  </packing>
-                </child>
               </object>
-              <packing>
-                <property name="position">1</property>
-              </packing>
             </child>
             <child>
-              <object class="GtkButton" id="btn_report">
-                <property name="label" translatable="yes">_Report</property>
-                <property name="related_action">gac_report</property>
+              <object class="GtkToggleButton" id="tbtn_search">
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
-                <property name="halign">end</property>
-                <property name="valign">end</property>
-                <property name="use_underline">True</property>
-                <style>
-                  <class name="suggested-action"/>
-                </style>
+                <property name="tooltip_text" translatable="yes">Show filter box</property>
+                <child>
+                  <object class="GtkImage">
+                    <property name="visible">True</property>
+                    <property name="icon_name">edit-find-symbolic</property>
+                  </object>
+                </child>
               </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">True</property>
-                <property name="pack_type">end</property>
-                <property name="position">4</property>
-              </packing>
             </child>
             <child>
-              <object class="GtkButton" id="btn_delete">
-                <property name="label" translatable="yes">_Delete</property>
-                <property name="related_action">gac_delete</property>
+              <object class="GtkToggleButton" id="tbtn_multi_select">
```

Comment:
This button should definitely have a tooltip text (i.e.: `Select multiple problems`).